### PR TITLE
Add get started page

### PR DESCRIPTION
**Pull request by lfdebrux**
### What problem does this pull request solve?

Trello card: https://trello.com/c/4p6qkzvc/ 

We want a get started page explaining to users how to create an account and start using GOV.UK Forms. This PR adds the page designed by @hannahkc.

### Things to consider when reviewing

- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

<details><summary><h3>Screenshot</h3></summary>

Screenshot of `/get-started`

</details>

### CODE REVIEW

**Comment by hannahkc** [comment]
The content in the screenshot looks perfect.

File: app/views/pages/get_started.html.erb [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<% set_page_title("") %>
```
Comment (thomasiles) [inline app/views/pages/get_started.html.erb:1]: Is this supposed to be blank?

**Review by thomasiles — APPROVED** [review]
Looks good 👍

File: app/views/pages/get_started.html.erb (hunk 1 above)
Comment (lfdebrux) [reply app/views/pages/get_started.html.erb:1]: Woops, no, I forgot to go back and check what the page title is supposed to be. Thanks for catching that!

@hannahkc do you have a page title in mind for this page? I couldn't see one on the google doc.

**Comment by hannahkc** [comment]
Same pattern as the other pages I reckon? So: Get started - GOV⁠.⁠UK Forms

That sound ok?

File: app/views/pages/get_started.html.erb (hunk 1 above)
Comment (hannahkc) [reply app/views/pages/get_started.html.erb:1]: Same pattern as the other pages I reckon? So: Get started - GOV⁠.⁠UK Forms

That sound ok?

Comment (lfdebrux) [reply app/views/pages/get_started.html.erb]: Done!

**Review by thomasiles — APPROVED** [review]
nice

**Pull request merged**